packages/govern-token/contracts/GovernTokenFactory.sol

```diff
@@ -37,7 +37,7 @@ contract GovernTokenFactory {
         GovernMinter minter
     ) {
         if (!_useProxies) {
-            (token, minter) = _deployContracts(_initialMinter, _tokenName, _tokenSymbol, _tokenDecimals);
+            (token, minter) = _deployContracts(address(this), _tokenName, _tokenSymbol, _tokenDecimals);
```

Comment:
Thanks for sorting this out! :) 

We can remove the ``_initialMinter`` argument from the ``_deployContracts`` method at all and just use ``address(this)`` directly in the ``_deployContracts`` method. This will for sure save a bit of gas.

Another problem we have here is that in the end the ``GovernBaseFactory`` is the defined minter. See lines 65-66 here and line 50 in ``GovernBaseFactory``. I think the goal would have been to grant the permissions to ``_mintAddr`` and not ``_initialMinter``. @novaknole Thanks for asking back related to the second problem we have here.